integration/simulation/utils.go

```diff
@@ -88,6 +90,23 @@ func getRollupHeader(client rpc.Client, hash gethcommon.Hash) *common.Header {
 	return result
 }
 
+// Uses the client to retrieve the current block number.
+func getBlockNumber(client rpc.Client) uint64 {
```

Comment:
Can we add the method to Obsclient and remove this?

The analogous gethclient method looks like this:
```
// BlockNumber returns the most recent block number
func (ec *Client) BlockNumber(ctx context.Context) (uint64, error) {
	var result hexutil.Uint64
	err := ec.c.CallContext(ctx, &result, "eth_blockNumber")
	return uint64(result), err
}
```

Reply:
Some of the other methods in here should be on the client too, could maybe leave a todo for them